### PR TITLE
Added a Time Travel Feature and updated Readme.md file

### PR DESCRIPTION
**Pull request by Dustin-Layne**
I added a method to fast forward time by roughly 3 hours when the user pushes the button 'y'. This will allow the user to quickly switch to day or night time based on their playing preference.

I also added a method to send the sun backwards by roughly 3  hours so that the player can reverse time if they want to.

I updated the readme.md file to be uptodate on current keybinds.

### CODE REVIEW

File: src/config.h [hunk 1]
```diff
@@ -45,6 +45,8 @@
 #define CRAFT_KEY_SIGN '`'
 #define CRAFT_KEY_AUTOWALK 'Q'
 #define CRAFT_KEY_PAUSEMENU 'M'
+#define CRAFT_KEY_TIMETRAVEL 'Y'
```
Comment (jrichards1999) [inline src/config.h:48]: Good use of descriptive variable names here to make the code more readable.

File: src/main.c [hunk 2]
```diff
@@ -2392,6 +2392,12 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
                 g->isWalking = false;     
 			} 
 		}
+        if (key == CRAFT_KEY_TIMETRAVEL) {
+            glfwSetTime(glfwGetTime() + 80);
```
Comment (jrichards1999) [inline src/main.c]: Code is easy enough to understand here, but I'd recommend setting the time interval to a constant variable and using that instead so the reader knows what 80 means.

**Review by jrichards1999 — COMMENTED** [review]
Dustin, the code is simple and well implemented.  I suggest adding some test code like print statements to the console that say something like "Skipping ahead in time" or "Rewinding time".

**Comment by nadamswsu** [comment]
good use of variable names to make the code easier to read and this can definitely use some test code to show that the program works by outputting to the console as a form of testing and documentation.

**Comment by w050asg** [comment]
Looks good pretty straightforward code that has good variable names. I agree that this code would be a really good candidate for testing.

**Pull request merged**